recipes-kernel/linux/5.15/patches/blktap2.patch

```diff
@@ -642,6 +643,19 @@ PATCHES
 +
 +#define dev_to_blktap(_dev) container_of(_dev, struct blktap, device)
 +
++static struct block_device *bdget_disk(struct gendisk *disk, int partno)
++{
```

Comment:
bdget_disk was removed in 5.15, so you had to re-add it?

Reply:
Good call.